fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/RuntimeFilterTranslator.java

```diff
@@ -84,6 +84,8 @@ private class RuntimeFilterExpressionTranslator extends ExpressionTranslator {
 
         @Override
         public Expr visitSlotReference(SlotReference slotReference, PlanTranslatorContext context) {
+            slotReference = context.getRuntimeTranslator().get()
+                    .context.getCorrespondingOlapSlotReference(slotReference);
```

Comment:
rebase to newest master in case of repeat these codes twice after merge into master